sherpa/astro/utils/tests/test_smoke.py

```diff
@@ -17,7 +17,11 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
-import mock
+try:
```

Comment:
Would it be better to follow https://github.com/sherpa/sherpa/blob/master/sherpa/ui/tests/test_session.py#L32 and import the python 3 version first - e.g.

```
try:
    from unittest import mock
except ImportError:
    import mock
```

(then when we remove python 2 support it's hopefully easier not to make a mistake!)

Reply:
It would have been slightly better (but then again we could be even better off using the `pytest-mock` plugin, or using `six.moves`, or something like that, but I think we are OK.